src/live/emitters.js

```diff
@@ -0,0 +1,5 @@
+function sendOrderCancelledEvent(io, payload) {
+  io.to(payload.userId).emit('orders:cancelled', payload.message);
```

Comment:
Instead of asking the caller to import `io` and pass it to this function; I think it makes more sense to have this file/module import `io` itself. Since this is where all the socket interaction is happening.

```suggestion
const { io } = require("../expressServer");

function sendOrderCancelledEvent(payload) {
  io.to(payload.userId).emit('orders:cancelled', payload.message);
```